examples/lwm2m/README.md

```diff
@@ -111,3 +111,7 @@ BOARD=<board> make clean all flash term
 #### Shell commands
```

Comment:
Because...github, I can´t comment on the correct line - I'm referring to L22-28. Could you split that up into:
```shell
wget https://hudson.eclipse.org/leshan/job/leshan/lastSuccessfulBuild/artifact/leshan-server-demo.jar
```
...and run the jar:
```shell
java -jar ./leshan-server-demo.jar
```

That way the copy sign works as expected in the UI of GitHub.

Reply:
Adapted